examples/react/playground/package.json

```diff
@@ -32,10 +32,10 @@
     "@storybook/testing-library": "^0.0.13",
     "@types/react": "^18.0.26",
     "@types/react-dom": "^18.0.9",
-    "@ucanto/interface": "^4.0.3",
+    "@ucanto/interface": "^4.2.3",
```

Comment:
I think it would be a good idea for `@w3ui/react`  and perhaps others from the family to re-export same basics like `@ucanto/interaface` and perhaps some things from `multiformats` so that they don't need to be imported by the users and deal with dependency problems.